accelerator/main.c

```diff
@@ -135,8 +135,7 @@ int main(int argc, char* argv[]) {
   }
 
   // Once tangle-accelerator finished initializing, return 'SIGUSR1' to parent process
```

Comment:
Since you no longer depends on `SIGUSR1`, the comment should be revised accordingly.